src/CoreWCF.Primitives/src/CoreWCF/Description/ServiceEndpoint.cs

```diff
@@ -37,7 +37,7 @@ public ServiceEndpoint(ContractDescription contract, Binding binding, EndpointAd
             get { return Behaviors; }
         }
 
-        internal KeyedByTypeCollection<IEndpointBehavior> Behaviors
+        public KeyedByTypeCollection<IEndpointBehavior> Behaviors
         {
```

Comment:
Similar to OperationDescription, leave internal and use the EndpointBehaviors property which you can cast if you need the Find method.

Reply:
WebHttp also needs to add behaviors here in WebHttpServiceBehavior and WebHttpEndpoint.

Reply:
That's fine, they can still do that, either by getting the collection and casting it, or using one of the insert/adding methods on KeyedCollection.

Reply:
This is addressed in 1a964b04c013f05f805dadc28e8741605ecfb71b.